app/src/main/java/com/example/paulsuarez/a28_filterable_lists/DetailActivity.java

```diff
@@ -0,0 +1,10 @@
+package com.example.paulsuarez.a28_filterable_lists;
+
+import android.support.v4.app.Fragment;
+
+public class DetailActivity extends RosterActivity{
```

Comment:
This might be better extended from ThemeHeaderFragmentActivity, as it isn't really very related to the List (at least from a class inheritance standpoint).